pytest/tests/sanity/gc_after_sync.py

```diff
@@ -16,15 +16,16 @@
 AFTER_SYNC_HEIGHT = 150
 TIMEOUT = 300
 
-consensus_config = {
+node_config = {
     "consensus": {
         # We're generating 150 blocks here - lower the min production delay to speed
         # up the test running time a little.
         "min_block_production_delay": {
             "secs": 0,
             "nanos": 100000000
         },
-    }
+    },
+    "state_sync_enabled": True
```

Comment:
Minor idea: We can add a comment here mentioning that state sync is necessary for this test and explaining why (or providing a link to the explanation) if it's not obvious.